core/kernel.rbs

```diff
@@ -1728,7 +1748,7 @@ module Kernel : BasicObject
   # :experimental
   # :   Used for experimental features that may change in future releases.
   #
-  def self?.warn: (*_ToS msg, ?uplevel: int?, ?category: Warning::category?) -> nil
+  def self?.warn: (*_ToS msgs, ?uplevel: int?, ?category: Warning::category | _ToSym | nil) -> nil
```

Comment:
One of the few places in the stdlib that uses `_ToSym` and not `interend`; the return value should be a `Warning::category`